tools_iuc.yaml

```diff
@@ -3811,6 +3811,9 @@ tools:
     owner: iuc
     tool_panel_section_label: Quality Control
 
+ -  name: fbqs_integrator
+    owner: SPAMR-VET
```

Comment:
This file is about IUC tools. These are tools maintained and reviewed by a large community at: https://github.com/galaxyproject/tools-iuc

You are welcome to submit your tool there if you like.

If you have it somewhere else you can add your tools to this file: https://github.com/usegalaxy-eu/usegalaxy-eu-tools/blob/master/diverse.yaml